satpy/composites/__init__.py

```diff
@@ -555,26 +555,29 @@ def _insert_palette_colors(channels, palette):
 class DayNightCompositor(GenericCompositor):
     """A compositor that blends a day data with night data."""
 
-    def __init__(self, name, lim_low=85., lim_high=88., **kwargs):
+    def __init__(self, name, lim_low=85., lim_high=88., chose_day_night="day_and_night", **kwargs):
         """Collect custom configuration values.
 
         Args:
             lim_low (float): lower limit of Sun zenith angle for the
                              blending of the given channels
             lim_high (float): upper limit of Sun zenith angle for the
                              blending of the given channels
+            chose_day_night (string): "day_and_night" means both day and night parts will be kept
```

Comment:
What do you think about `day_night` for the keyword argument (otherwise is should be `choose_` not `chose_`)?

Reply:
Sorry for the spell...